### ISSUE TITLE
Actual Logo -> Post-Logo Design Thread

### ISSUE THREAD

**Issue by JohnAllen** [Distense/distense-ui#85]

**Comment by JohnAllen** [comment]
For this, we should have a competition ideally.  There's one task for this -- can have multiple PRs and people wait to approve the PRs in the client until they love a logo.

**Comment by e-n-l** [comment]
I'd like to take a shot at this, but has there been any conversation yet about what you're looking for in an "actual logo"?

**Comment by JohnAllen** [comment]
Really appreciate you getting in touch about this, @razordaze!  Fortunately/unfortunately we got an actual logo 3 days ago and I haven't yet closed this issue (sorry).

That said, I don't want to call it quits on design at all, and actually was about to contact someone about doing some illustrations for our landing page that will hopefully help people more quickly grasp what Distense is.  

I have also pondered tweaking our new-found logo (check it [here](https://twitter.com/distenseorg)) if you wish, perhaps with a black circle around it.  It's unlikely we're currently working with the best version of it.

I also want to create numerous graphics, akin to our pinned tweet (again, [here](https://twitter.com/distenseorg)), that at least adhere to a consistent font, and font-color.

So if there are other ways you'd like to help, feel free to propose stuff or pick one!  [Our site](https://disten.se) could still use quite a bit of help in general as well (we haven't even updated it yet)!

And in case you're interested in the illustrations: one metaphor for Distense is a giant office building (I like to think of the Empire State Building), that typically would have executives making the decisions on the top floor.  But instead Distense has computers carrying out the executive functions instead. So the idea is to have two illustrations: one with the executives on the top floor and one with a bunch of computer looking things.

yet another cool thing to have would be a Twitter cover that doesn't just say "Distense", like it does now, but uses our green in a creative way. The node/graph theme is super worn out in the crypto world and I don't have any specific ideas there yet though...
